src/App.js

```diff
@@ -0,0 +1,21 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Books from './components/Books';
+import Categories from './components/Categories';
+import Navbar from './components/Navbar';
+import Form from './components/Form';
+import './app.css';
+
+function App() {
+  return (
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route exact path="/" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+      <Form />
+    </Router>
+  );
```

Comment:
- [ ] Kindly render the `Form` component in the `Books` component. Remember that the form is used to add new books and only the `Books` page should know about the form if you follow the separation of concern principle. Kindly implement this change. 👍